core/src/main/java/hudson/cli/ReloadJobCommand.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2015 Red Hat, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package hudson.cli;
+
+import hudson.Extension;
+import hudson.cli.handlers.ViewOptionHandler;
+import hudson.model.AbstractItem;
+import hudson.model.AbstractProject;
+import hudson.model.ViewGroup;
+import hudson.model.View;
+
+import jenkins.model.Jenkins;
+import org.kohsuke.args4j.Argument;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.logging.Logger;
+
+/**
+ * @author pjanouse
+ * @since TODO
+ */
+@Extension
+public class ReloadJobCommand extends CLICommand {
+
+    @Argument(usage="Name of the job(s) to reload", required=true, multiValued=true)
+    private List<String> jobs;
+
+    private static final Logger LOGGER = Logger.getLogger(ReloadJobCommand.class.getName());
+
+    @Override
+    public String getShortDescription() {
+
+        return Messages.ReloadJobCommand_ShortDescription();
+    }
+
+    @Override
+    protected int run() throws Exception {
+
+        boolean errorOccurred = false;
+        final Jenkins jenkins = Jenkins.getInstance();
+
+        if (jenkins == null) {
+            stderr.println("The Jenkins instance has not been started, or was already shut down!");
+            return -1;
+        }
+
+        final HashSet<String> hs = new HashSet<String>();
+        hs.addAll(jobs);
+
+        for (String job_s: hs) {
+            AbstractItem job = null;
+
+            try {
+                job = (AbstractItem) jenkins.getItemByFullName(job_s);
```

Comment:
But Item (Interface) doesn't declare any useful 'reload' behavior. I don't want to duplicate a code from AbstractProject.doReload() here. 

Nobody can rely on reload from Item because it doesn't exists - so I can't break a backward compatibility as well.